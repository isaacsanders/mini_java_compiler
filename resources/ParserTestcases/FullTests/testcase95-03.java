class BadClass {
    public static void main(String[] args) {
        System.out.println();
        
        AnotherBadClass a = new AnotherBadClass(one());
    } 
    
    public int one() {
        return 1;
    }
}

class AnotherBadClass {
    Integer a = new Integer();
    
    public int f(int x, int y) {
        return x + y;
    }
    
    int c;
}