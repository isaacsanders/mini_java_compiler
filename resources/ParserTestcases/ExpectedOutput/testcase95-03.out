StmtList ::= 


ERROR in line 3 and column 28: Syntax error

Error recovery to next semicolon.
StmtList ::= StmtList Stmt
Type ::= ID


ERROR in line 5 and column 49: Syntax error

Error recovery to next semicolon.
StmtList ::= StmtList Stmt
MainClassDecl ::= ...
ClassDeclList ::= 
OptExtends ::= 
ClassVarDeclList ::= 
Type ::= ID


ERROR in line 14 and column 15: Syntax error



ERROR in line 14 and column 15: Couldn't repair and continue parse

Exception processing file: Can't recover from previous error(s)
